dashboard/src/main.ts

```diff
@@ -50,9 +50,9 @@ app.config.globalProperties.$filters = {
       | api.EnduroStorageAipWorkflowTypeEnum,
   ) {
     switch (value) {
-      case api.EnduroIngestSipWorkflowTypeEnum.CreateAip:
+      case api.EnduroIngestSipWorkflowTypeEnum.Aip:
         return "Create AIP";
-      case api.EnduroIngestSipWorkflowTypeEnum.CreateAndReviewAip:
+      case api.EnduroIngestSipWorkflowTypeEnum.AndReviewAip:
```

Comment:
Don't ask me why ;)

I tried re-generating the client code more than once, but those are the types it chooses.

Reply:
I figured this out: the OpenAPI generator defaults to removing identical prefixes from enum names ("removeEnumValuePrefix").  I created a new branch with the fix so I didn't mess with your PR: https://github.com/artefactual-sdps/enduro/compare/dev/issue-1218-fix-dashboard-workflow-types

Feel free to merge the fix if that works for you.

Reply:
Niiice, thanks for the detective work!